packages/klighd-core/src/options/render-options-registry.ts

```diff
@@ -71,11 +117,13 @@ export class ShowConstraintOption implements RenderOption {
  */
 export class UseSmartZoom implements RenderOption {
     static readonly ID: string = 'use-smart-zoom'
-    static readonly NAME: string = 'Smart Zoom'
+    static readonly NAME: string = 'Enable Smart Zoom'
```

Comment:
I think in the semantics it was discussed at some point that no options should have "show", "enable" or similar in their option label, as having the checkbox should clarify that this option is enabled when it is checked. Furthermore, the description should now further clarify that if the user does not immediately understand what the option does from its name.

Reply:
Ok